sklearn/impute.py

```diff
@@ -825,8 +846,13 @@ def fit_transform(self, X, y=None):
                                      check_random_state(self.random_state))
 
         if self.predictor is None:
-            from .linear_model import BayesianRidge
-            self._predictor = BayesianRidge()
+            if self.sample_posterior:
+                from .linear_model import BayesianRidge
+                self._predictor = BayesianRidge()
+            else:
+                from .linear_model import RidgeCV
+                # including a very small alpha to approximate OLS
+                self._predictor = RidgeCV(alphas=np.array([1e-5, 0.1,  1, 10]))
```

Comment:
You will need to set the random state of the estimator.

Reply:
if `random_state is not None`

Reply:
I'm confused, there is no `random_state` anywhere here: http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.RidgeCV.html

Reply:
Uhm. I have to check the code. I could have think that the CV was actually relying on that but I might be wrong.

Reply:
I looked the `RidgeCV` code - no trace of random state!